.github/workflows/docs.yml

```diff
@@ -158,6 +176,25 @@ jobs:
           publish_dir: doc/_build/html/
           cname: docs.pyvista.org
 
+      - name: Remove non-multi version folder
+        run: rm -rf doc
+
+      - name: Deploy on main multiversion
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: pyansys/actions/doc-deploy-dev@main
+        with:
+            cname: ${{ env.DOCUMENTATION_CNAME }}
```

Comment:
Reminder, just to not forget to add this environment variable on top of this file.